[2023-11-22T05:25:54.466+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.Create_source_table manual__2023-11-22T05:25:51.987352+00:00 [queued]>
[2023-11-22T05:25:54.489+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.Create_source_table manual__2023-11-22T05:25:51.987352+00:00 [queued]>
[2023-11-22T05:25:54.489+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-22T05:25:54.490+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2023-11-22T05:25:54.490+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-22T05:25:54.525+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): Create_source_table> on 2023-11-22 05:25:51.987352+00:00
[2023-11-22T05:25:54.536+0000] {standard_task_runner.py:55} INFO - Started process 5264 to run task
[2023-11-22T05:25:54.542+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rand_user', 'Create_source_table', 'manual__2023-11-22T05:25:51.987352+00:00', '--job-id', '657', '--raw', '--subdir', 'DAGS_FOLDER/predictions.py', '--cfg-path', '/tmp/tmpg0qqam_8']
[2023-11-22T05:25:54.546+0000] {standard_task_runner.py:83} INFO - Job 657: Subtask Create_source_table
[2023-11-22T05:25:54.683+0000] {task_command.py:376} INFO - Running <TaskInstance: rand_user.Create_source_table manual__2023-11-22T05:25:51.987352+00:00 [running]> on host 91f7624ceef9
[2023-11-22T05:25:54.866+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Gaurav
AIRFLOW_CTX_DAG_ID=rand_user
AIRFLOW_CTX_TASK_ID=Create_source_table
AIRFLOW_CTX_EXECUTION_DATE=2023-11-22T05:25:51.987352+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-22T05:25:51.987352+00:00
[2023-11-22T05:25:54.896+0000] {base.py:71} INFO - Using connection ID 'source_db_***' for task execution.
[2023-11-22T05:25:54.926+0000] {sql.py:315} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS rand_users (
        
        uuid TEXT PRIMARY KEY,
        title TEXT NOT NULL,
        firstname TEXT NOT NULL,
        lastname TEXT NOT NULL,
        gender TEXT NOT NULL,
        country TEXT NOT NULL,
        username TEXT NOT NULL,
        password TEXT NOT NULL,
        email TEXT NOT NULL,
        age TEXT NOT NULL
        )
        , parameters: None
[2023-11-22T05:25:54.935+0000] {postgres.py:96} INFO - NOTICE:  relation "rand_users" already exists, skipping

[2023-11-22T05:25:54.967+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=rand_user, task_id=Create_source_table, execution_date=20231122T052551, start_date=20231122T052554, end_date=20231122T052554
[2023-11-22T05:25:55.035+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-22T05:25:55.130+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
